### PR TITLE
Added template expansion for helm chart version (#5709)

### PR DESCRIPTION
**Pull request by kadern0**
Signed-off-by: Pablo Caderno <kaderno@gmail.com>

Fixes: #5709  
**Related**: _Relevant tracking issues, for context_
**Merge before/after**: _Dependent or prerequisite PRs_

**Description**

Added template expansion for Helm chart version. 

Before (no variable expansion):

initial

After (it uses the value):
final

If variable is not defined:
Screenshot from 2021-07-08 08-35-10

### CODE REVIEW

**Review by briandealwis — APPROVED** [review]
Thanks @kadern0!  I'll make the change I suggested below and added a quick test for verification.

**Comment by briandealwis** [comment]
@googlebot I consent.

File: pkg/skaffold/deploy/helm/args.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ type installOpts struct {
 	helmVersion  semver.Version
 	postRenderer string
 	repo         string
+	Version      string
```
[the remark this answers is not in the document]
Comment (kadern0) [reply pkg/skaffold/deploy/helm/args.go]: You're right.

**Comment by kadern0** [comment]
> Thanks @kadern0! I'll make the change I suggested below and added a quick test for verification.

Thanks :)

**Pull request merged**